modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -770,7 +777,7 @@ ALTER USER name [IF EXISTS]
   [[SET PASSWORD] CHANGE [NOT] REQUIRED]
   [SET STATUS {ACTIVE | SUSPENDED}]
   [SET HOME DATABASE name]
-  [SET AUTH [PROVIDER] 'provider' "{"SET <key> <value>"}"]
+  {[SET AUTH [PROVIDER] 'provider' "{"{SET <key> <value>}..."}"]}...
   [REMOVE HOME DATABASE]
   [REMOVE { AUTH [PROVIDER[S]] provider[, ...] } \| { ALL AUTH [PROVIDER[S]] }]
```

Comment:
missed updating this one (as my suggestion was only given on the first one I assume), also added the repeating here
```suggestion
  [REMOVE { AUTH [PROVIDER[S]] provider[, ...] \| ALL AUTH [PROVIDER[S]] }]...
```
or to have the extra brackets like SET
```suggestion
  {[REMOVE { AUTH [PROVIDER[S]] provider[, ...] \| ALL AUTH [PROVIDER[S]] }]}...
```